RecoMuon/L2MuonSeedGenerator/plugins/L2MuonSeedGeneratorFromL1TkMu.cc

```diff
@@ -311,14 +369,14 @@ void L2MuonSeedGeneratorFromL1TkMu::produce(edm::Event &iEvent, const edm::Event
                 container.push_back(*tsci);
```

Comment:
Just in case, in the master branch, there is a range based loop here, https://github.com/cms-sw/cmssw/pull/32713/files#diff-70b2a719aac32ca979c0ac376047b49c9ed92e9098d0eca81fb05ea80b6ae16dR367-R369
```
              for (auto const &recHit : assoOffseed->recHits()) {
                container.push_back(recHit);
              }
```
so the backport and the original PR differ. Let me know if this was intentional.

Reply:
Yes it's intended. The TrajectorySeed class was updated to use edm::Range in 11_3:
https://github.com/cms-sw/cmssw/commit/877108fae02ea1fec71cf736b464a72b151d1658#diff-e4ad11aac7f1defe33fb2758bc5ae0ad95b2eb4964cc761285b22266f8e13d67L50